src/Tribe/Views/V2/Hooks.php

```diff
@@ -138,6 +142,15 @@ public function action_disable_shortcode_assets_v1() {
 		$assets->disable_v1();
 	}
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$wp_admin_bar" missing

<sub>hash: 60a1db76fa11a72a49ded6985582b236</sub>